core/integration/services_test.go

```diff
@@ -1244,7 +1244,7 @@ mkdir srv
 cd repo
 	git init
 	git branch -m main
-	git add *
+	git add * || true
```

Comment:
I don't think this is needed anymore; fixed it as part of #5474

Reply:
+1